framework/wazuh/core/cluster/master.py

```diff
@@ -386,8 +387,13 @@ def hello(self, data: bytes) -> Tuple[bytes, bytes]:
 
         # SyncFiles instance used to zip and send integrity files to worker.
         self.integrity = c_common.SyncFiles(cmd=b'syn_m_c', logger=self.task_loggers['Integrity sync'], manager=self)
-        # Initialize agent-groups sending task.
-        self.agent_group_task = asyncio.ensure_future(self.send_agent_groups_information())
```

Comment:
The `self.agent_group_task` should also be removed from the `__init__` and  `connection_lost` methods since it is not being set anymore.

Reply:
Done in https://github.com/wazuh/wazuh/pull/13906/commits/774230984783d91d27044c13f647ee9af920eb3f